DiamondLightSource/python-artemis
# (#286) Make wrong filename PVs read only

Fixes #286 

To test:
* Confirm unit tests pass
* Try setting the wrong PV (e.g. add `self.odin.meta.file_name.put(file_prefix)` to `set_odin_pvs`) and confirm unit tests would then fail
